docs/en/02_Developer_Guides/14_Files/03_File_Security.md

```diff
@@ -352,11 +359,70 @@ SilverStripe\Filesystem\Flysystem\FlysystemAssetStore:
     Pragma: 'no-cache'
 ```
 
+### Configuring file HTTP response code
+
+When a user tries to access a file that exists, but for which they do not have access,
+Silverstripe CMS will return a "404 Not found" response rather than a "403 Denied" to
+avoid revealing the existence of the file.
+
+You can customise the response codes for various types of request via
+configuration flags on `FlysystemAssetStore`.
+
+```yml
+# Add this code snippet to your YML configuration
```

Comment:
```suggestion
```

I don't think we need to say things like this, there are YAML snippets throughout our documentation - devs are pretty familiar with what it is by the point they're this deep in our docs.

Reply:
Fixed.